gammapy/analysis/tests/test_analysis.py

```diff
@@ -175,8 +134,8 @@ def test_analysis_1d_stacked():
     assert_allclose(analysis.datasets["stacked"].counts.data.sum(), 404)
     pars = analysis.fit_result.parameters
 
-    assert_allclose(pars["index"].value, 2.689559, rtol=1e-3)
-    assert_allclose(pars["amplitude"].value, 2.81629e-11, rtol=1e-3)
+    assert_allclose(pars["index"].value, 2.676283, rtol=1e-3)
+    assert_allclose(pars["amplitude"].value, 4.554215e-11, rtol=1e-3)
```

Comment:
Why did the flux amplitude increase by a factor of 2 here?
Ideally this PR would just change to the new code, but keep the existing analysis tests the same.